src/lib/pages/backoffice/Item/ItemForms/ItemEditor/ItemForm/ItemForm.js

```diff
@@ -99,8 +99,15 @@ export class ItemForm extends Component {
       successSubmitMessage,
       title,
       pid,
+      itemPastLoans,
     } = this.props;
     const initialValues = data ? this.prepareData(metadata) : {};
+    const hasPastOrActiveLoans =
+      itemPastLoans > 0 ||
+      (data.metadata.circulation &&
+        data.metadata.circulation.state === 'ITEM_ON_LOAN');
```

Comment:
Really sorry to comment again @jrcastro2 :( but it is maybe safer to do:
```suggestion
        invenioConfig.CIRCULATION.loanActiveStates.includes(data.metadata.circulation.state);
```